### ISSUE TITLE
Bug: pressing escape on quick map page deletes the sidebar

### ISSUE THREAD

**Issue by Fuhji** [micronutrientsupport/micronutrient-support-tool#523]
## Description
On the quick-maps page, when pressing the escape key the sidebar is deleted and the map maximises.

## Expected Behaviour
Any open dialogs or forms should be dismissed, but otherwise the sidebar and map should not be affected
## Actual Behaviour
The sidebar is deleted.

## Possible Fix
```
Not obligatory, but suggest a fix or reason for the bugissue
```

## Steps to Reproduce
1. go to quick-maps page

![image](https://user-images.githubusercontent.com/4210022/121361882-91195580-c92d-11eb-9be4-618746788105.png)

2. click the sidebar
3. press escape
![image](https://user-images.githubusercontent.com/4210022/121361983-a8584300-c92d-11eb-98d7-caa441dbe22f.png)

## Your Environment

* Version used: 
* Browser Name and version: chrome
* Operating System and version (desktop or mobile): mac os x desktop
* Link to your project:

**Comment by DanielLyleWilliamWarren** [comment]
https://github.com/angular/components/issues/6393 

Issue has been highlighted by other develops, looking at a fix now

**Comment by DanielLyleWilliamWarren** [comment]
Resolved with MR #527 , Closing issue.
